cmd/internal/loginprovider/sessiontokenloginprovider.go

```diff
@@ -55,6 +55,11 @@ type sessionTokenLoginProvider struct {
 	updateAccountDetailsFunc func(string) error
 }
 
+// Token implements the LoginProvider.Token method.
```

Comment:
Thanks for the comment but can we add some more depth to the comment explaining what this provider does when the func is called?